assets/stylesheets/screens/_keyword.scss

```diff
@@ -1,9 +1,16 @@
 body.keyword {
-  .form-keyword-upload {
+  .form-keyword-upload,
+  .form-keyword-filter {
     @apply border rounded-md p-6;
     width: fit-content;
```

Comment:
Not related to this PR, but you can customize the width in `tailwind.config.js file`, then you can re-use it
https://v1.tailwindcss.com/docs/width#customizing
```
// tailwind.config.js
module.exports = {
    theme: {
      extend: {
        width: {
           "fit-content": "fit-content";
        }
      }
    }
  }

// Then you can use:
@apply w-fit-content;
```

Reply:
That's look greats. So I can add any style I want into it, right? such as sometimes they don't provide `w-11`.